openslides/core/static/js/core/site.js

```diff
@@ -107,6 +107,38 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+// Set up the activeApp for the title from the webpage
+.run([
+    '$rootScope',
+    'gettextCatalog',
+    function ($rootScope, gettextCatalog) {
+        $rootScope.activeApp = '';
+        $rootScope.$on('$stateChangeSuccess', function(evt, toState) {
+            console.log(toState);
+            if (toState.name.startsWith('core.tag')) {
+                $rootScope.activeApp = gettextCatalog.getString('Tags');
+            } else {
+                var state = toState.name.split('.')[0];
+                switch (state) {
```

Comment:
I think not. You may have a look at the mainMenu provider. May be this is easily extendable. Or we extend the $stateProvider and set the new verbose app name in lines like motions/site.js:31. This would be better than this extra hard coded map of app name – app label strings.